cli.js

```diff
@@ -28,6 +29,12 @@ const cli = meow(`
 
 updateNotifier({pkg: cli.pkg}).notify();
 
+const pkg = readPkgUp.sync().pkg;
```

Comment:
When I inspected the contents of `cli.pkg` it seemed to contain the package info for `np`, not the package you were trying to publish. I'll double-check that though :)